src/core/src/core_logic/AutoAssessmentConfigurator.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 2.7+
+
+from core.src.bootstrap.Constants import Constants
+
+class AutoAssessmentConfigurator(object):
+    def __init__(self, env_layer, execution_config, composite_logger, telemetry_writer, status_handler):
+        self.env_layer = env_layer
+        self.execution_config = execution_config
+
+        self.composite_logger = composite_logger
+        self.telemetry_writer = telemetry_writer
+        self.status_handler = status_handler
+
+
+    def create_configuration(self):
+        code, output = self.env_layer.run_command_output()
```

Comment:
which command is being executed here?

Reply:
I guess there should be at least one param here: run_command_output(self, cmd, no_output=False, chk_err=False)

Reply:
This entire class is dead code - it's not being used anywhere. It was part of a skeleton I initially wrote and got left behind. Thanks for catching it..